bouncer/shared/swapping.ts

```diff
@@ -15,56 +14,8 @@ import { BtcAddressType } from '../shared/new_btc_address';
 import { CcmDepositMetadata } from '../shared/new_swap';
 import { SwapContext, SwapStatus } from './swap_context';
 
-enum SolidityType {
-  Uint256 = 'uint256',
-  String = 'string',
-  Bytes = 'bytes',
-  Address = 'address',
-}
-
 let swapCount = 1;
 
-function newAbiEncodedMessage(types?: SolidityType[]): string {
-  const web3 = new Web3();
-
-  let typesArray: SolidityType[] = [];
-  if (types === undefined) {
-    const numElements = Math.floor(Math.random() * (Object.keys(SolidityType).length / 2)) + 1;
-    for (let i = 0; i < numElements; i++) {
-      typesArray.push(
-        Object.values(SolidityType)[
-          Math.floor(Math.random() * (Object.keys(SolidityType).length / 2))
-        ],
-      );
-    }
-  } else {
-    typesArray = types;
-  }
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const variables: any[] = [];
-
-  for (let i = 0; i < typesArray.length; i++) {
-    switch (typesArray[i]) {
-      case SolidityType.Uint256:
-        variables.push(randomAsNumber());
-        break;
-      case SolidityType.String:
-        variables.push(Math.random().toString(36).substring(2));
-        break;
-      case SolidityType.Bytes:
-        variables.push(randomAsHex(Math.floor(Math.random() * 100) + 1));
-        break;
-      case SolidityType.Address:
-        variables.push(randomAsHex(20));
-        break;
-      // Add more cases for other Solidity types as needed
-      default:
-        throw new Error(`Unsupported Solidity type: ${typesArray[i]}`);
-    }
-  }
-  return web3.eth.abi.encodeParameters(typesArray, variables);
-}
-
```

Comment:
I originally had this to check that abi encoded messages work but it's actually better to have generalize to any arbitrary bytes for this end to end testing.